api/src/api/language.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const router = express.Router();
+const { errorWrap } = require('../middleware');
+const { sendResponse } = require('../utils/response');
+const { models } = require('../models/index.js');
+const { requireAuthentication } = require('../middleware/authentication');
+const _ = require('lodash');
+/**
+ * Creates a new course in the database
+ *
+ * @param {newUser} New course
+ * @returns a new course under the given language
+ */
+router.post(
+  '/course',
+  requireAuthentication,
+
+  errorWrap(async (req, res) => {
+    const user = req.user;
+    const newCourse = new models.Course({
```

Comment:
code readability - can create a const variable called `courseData` and set it to `req.body.` Then, when instantiating a new course, you can do the following:

```
new models.Course({
approved: true,
admin_id: user.authID,
details: courseData.details});
```

Reply:
> Amazing work Jessica! Love to see the testing 😁
Thank you!!